lib/cn/csched_sp3.c

```diff
@@ -409,15 +409,23 @@ sp3_log_progress(struct cn_compaction_work *w, struct cn_merge_stats *ms, bool f
 
```

Comment:
Seems like we could clamp est->cwe_keys and avoid the if...else?

```
est->cw_keys = clamp_t(type, est->cw_keys, 1, est->cw_keys);
progress = 1.0 * ms->ms_keys_in / est->cwe_keys;
```

Reply:
Seems like a fine solution. I will do that.

Reply:
Implemented, but chose not to change est->cw_keys

Reply:
Why not just cast to double?

Reply:
with `1.0` stuff? I don't know. Didn't write the code. Want me to change it?

Reply:
Changed in latest push